tests/fixtures/envs/dummy/dummy_discrete_pixel_env.py

```diff
@@ -16,8 +16,8 @@ class DummyDiscretePixelEnv(DummyEnv):
     Several properties are made for testing purpose as following:
 
     -Observations are
-        after reset    : np.zeros(self._shape).
-        action 1 (FIRE): np.ones(self._shape).
+        after reset    : np.ones(self._shape).
```

Comment:
did this sneak in from a different PR?

Reply:
I changed it because we don't want to have zeros output